File: app/src/main/java/com/thetechnocafe/gurleensethi/bottomsheets/MainActivity.java
package com.thetechnocafe.gurleensethi.bottomsheets;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button bottomSheetsCoordinatorLayout = (Button) findViewById(R.id.bottomSheetsButton);
        Button bottomSheetsDialogFragment = (Button) findViewById(R.id.bottomSheetsDialogFragmentButton);

        bottomSheetsCoordinatorLayout.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(MainActivity.this, BottomSheetActivity.class);
                startActivity(intent);
            }
        });

        bottomSheetsDialogFragment.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(MainActivity.this, BottomSheetFragmentActivity.class);
                startActivity(intent);
            }
        });
    }
}

File: app/src/main/java/com/thetechnocafe/gurleensethi/bottomsheets/MainActivity.kt
package com.thetechnocafe.gurleensethi.bottomsheets

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Button
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        bottomSheetsButton.setOnClickListener {
            Intent(this@MainActivity, BottomSheetActivity::class.java).apply { startActivity(this) }
        }
        bottomSheetsDialogFragmentButton.setOnClickListener {
            Intent(this@MainActivity, BottomSheetFragmentActivity::class.java).apply { startActivity(this) }
        }
    }
}
